### ISSUE TITLE
Auto-completion for boolean labels after gateways

### ISSUE THREAD

**Issue by falko** [camunda/camunda-modeler#3053]
### Problem you would like to solve

A BPMN best practice is to label exclusive gateways with a question and it's outgoing sequence flows with the answers. The most common version of this pattern is to use yes/no questions.

![image](https://user-images.githubusercontent.com/40038/181014084-22771017-7d67-4c6c-b74e-7fe557c43078.png)

### Proposed solution

Can the Modeler automatically complete the sequence flow labels, i.e. if I type `Yes` or `No` on one sequence flow, the Modeler adds the opposite label to the other sequence flow.

### Alternatives considered

Manually adding the label

**Comment by marstamm** [comment]
Yes, this can be done with a plugin and I think this is also the correct place for this improvement.

I imagine it might be confusing for the user if labels are being added automatically when they did not enable it

**Comment by marstamm** [comment]
Might be a good topic for Slacktime/ Hackdays as well
